model/event/subscriptions_test.go

```diff
@@ -273,6 +273,61 @@ func (s *subscriptionsSuite) TestRegexSelectorsMatch() {
 	s.False(regexSelectorsMatch(selectors, a.RegexSelectors))
 }
 
+func (s *subscriptionsSuite) TestFilterFromSelectors() {
+	for _, sub := range s.subscriptions {
+		filter := Filter{}
+		s.NoError(filter.FromSelectors(append(sub.Selectors, sub.RegexSelectors...)))
```

Comment:
It seems like the assertion below this depends on this line succeeding. Should this be `s.Require().NoError`?

Reply:
done